src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    add(number, obj) {
```

Comment:
It's strange looking that these `add` `subtract` and so on methods take as an argument some `obj`
Let's will do that these methods will take **only** the number